synapsis/src/main/resources/templates/pages/login.html

```diff
@@ -1,20 +1,130 @@
+            <div class="flex-none w-1/2 mr-12">
+                <img src="" alt="Website Example">
+            </div>
+            <div class="flex-none container flex flex-wrap flex-col justify-center content-center mt-8 w-80 border rounded-sm">
+                <div class="mt-6">
+                    <img src="" class="w-8 inline" alt="S">
+                    <img src="/assets/synapsis_black.svg" class="inline h-10 w-48" alt="Synapsis Logo">
+                </div>
+                <input id="email" name="email" type="email" placeholder="Email"
+                       class="font-thin font-serif mt-4 pl-3 h-14 w-3/4 content-center border rounded-md"
+                       required/>
+                <input id="password" name="password" type="password" placeholder="Password"
+                       class="font-thin font-serif pl-3 mt-3 h-14 w-3/4 content-center border rounded-md"
+                       required/>
+
+                <button type="submit" class="btn btn-primary mt-3 bg-primary text-base justify-self-end">Login
+                </button>
+
+                <hr class="mt-3 text-black">
 
-        <label for="password">Password</label>
-        <input id="password" name="password" type="password" required />
+                <a class="mt-5 h-14 bg-facebook-blue rounded-md drop-shadow-md">
+                    <div class="mt-4">
+                        <img class="ml-3 mb-1 w-6 h-6 inline" src="/assets/facebook-logo.png">
+                        <div class="ml-2 mr-3 text-center text-white text-xl font-serif font-normal inline">Sign In with
+                            Facebook
+                        </div>
+                    </div>
+                </a>
+                <a class="mt-5 h-14 bg-white rounded-md drop-shadow-md">
+                    <div class="mt-4">
+                        <img class="ml-3 mb-1 w-6 h-6 inline" src="/assets/google-logo.png">
+                        <div class="ml-2 mr-3 text-center text-black text-xl font-serif font-normal inline">Sign In with
+                            Google
+                        </div>
+                    </div>
+                </a>
+                <a class="mt-5 h-14 bg-true-black rounded-md drop-shadow-md">
+                    <div class="mt-4">
+                        <img class="ml-4 mb-2 w-5 h-6 inline" src="/assets/apple-logo.png">
+                        <div class="ml-2 mr-3 text-center text-white text-xl font-serif font-normal inline">Sign In with
+                            Apple
+                        </div>
+                    </div>
+                </a>
+                <div class="mt-5 mb-4 text-black font-serif font-normal text-base ">
+                    Don't have an account? <a class="text-primary">Click here</a>
+                </div>
+            </div>
+        </div>
 
-        <button type="submit">Login</button>
     </form>
+
+    <div class="container flex flex-row justify-center items-center mt-48">
```

Comment:
Would this be better factored out as a fragment, since we may reuse it across different pages like logout and register?

Reply:
Yea we probably should do that